gst/vspfilter/gstvspfilter.c

```diff
@@ -64,7 +64,12 @@ enum
   PROP_OUTPUT_IO_MODE,
 };
 
-#define CSP_VIDEO_CAPS GST_VIDEO_CAPS_MAKE (GST_VIDEO_FORMATS_ALL)
+#define CSP_VIDEO_CAPS \
+    "video/x-raw, " \
+        "format = (string) {RGB16, RGB, BGR, ARGB, xRGB, BGRA, BGRx, I420, NV12, UYVY}," \
+        "width = [ 1, 8192 ], " \
+        "height = [ 1, 8192 ], " \
```

Comment:
Is the "(experimental)" tag here needed?  Offhand I can't see any reason why this shouldn't work. Have you noticed some potential problems with the NV16 support?